option optimize_for = LITE_RUNTIME;
import "charlie.proto";
package modules.manager;

message CManagerInfo
{
  //All of the possible onion root URLs
  repeated string onion_root = 1;
}
